src/ICeWall/doc/set_bump_options.md

```diff
@@ -0,0 +1,60 @@
+## set_bump_options
+
+### Synopsis
+```
+  % set_bump_options \
+        [-pitch pitch] \
+        [-spacing_to_edge spacing] \
+        [-bump_pin_name pin_name] \
+        [-cell_name bump_cell_table] \
+        [-num_pads_per_tile value] \
+        [-rdl_layer name] \
+        [-rdl_width value] \
+        [-rdl_spacing value] \
+        [-rdl_cover_file_name rdl_file_name]
```

Comment:
when using this option, we get a warning that `-rdl_cover_file_name` does not exist

Reply:
Fixed